cmake/Linux.cmake

```diff
@@ -68,6 +68,18 @@ file(COPY
   ${PROFINET_BINARY_DIR}/
   )
 
+file(COPY
```

Comment:
You can copy multiple files with one file(COPY), as long as the destionation is the same. Also, I think we should use install() here. The main difference between install() and file(COPY) is that the copy takes place at configuration time (when running cmake). If you wanted to install into /usr/local or similar you'd probably need to be root or similar just to run cmake. Whereas with install() you'd only need root for make install.

Reply:
Done. Also adjusted the tests now that #63 have been merged.

Reply:
Changed back to file(COPY) after discussions.